### PR TITLE
Add Fleet/Agent 8.6.1 relnotes

### LINKED ISSUE #2537: [REQUEST]: Add Fleet/Agent release notes for 8.6.1
### Description

Release is coming up and we need to make sure release notes get published.

### Resources

Docs tracking issue: https://github.com/elastic/dev/issues/2195

### Collaboration

Other (please describe below in the Point of contact section).

### Point of contact.

**Main contact:** @cmacknz 

@cmacknz Adding you as a contact because I think the release notes process for Elastic Agent still need some fine tuning and manual work. I'm assuming that:

* Someone on the dev team will generate the raw asciidoc file that we'll use as source for the Elastic Agent sections of the release notes.
* Someone on the writing team will create the Fleet and Fleet Server sections manually by looking at the fragments in the fleet-server repo and also looking at Fleet entries in the the Kibana release notes.

Is that still correct?

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #2537

Adds release notes compiled manually from fleet-server changelog, Fleet section in Kibana release notes, and elastic-agent fragments.

### CODE REVIEW

**Comment by dedemorton** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 1]
```diff
@@ -14,13 +14,40 @@
...
 
+* <<release-notes-8.6.1>>
 * <<release-notes-8.6.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.6.1 relnotes
```
Comment (cmacknz) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:25]: The known issue for Osquery results being delayed needs to carry into 8.6.1, we didn't have enough time to fix it for this release: https://github.com/elastic/observability-docs/blob/9e5574bc06839b9d49a01e1b6f852f13d7bcaff4/docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc?plain=1#L69-L85

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 2]
```diff
@@ -14,13 +14,58 @@
...
+
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:26]: There are some additional items that need to be incorporated. Here is the raw file:

// begin 8.6.1 relnotes

[[release-notes-8.6.1]]
==  8.6.1

Review important information about the  8.6.1 release.





[discrete]
[[known-issues-8.6.1]]
=== Known issues

// TODO: add details and impact

Fleet-Server::

[discrete]
[[known-issue-issue-https://github.com/elastic/elastic-agent/issues/2170]]
.Adding A Fleet Server Integration To An Agent That Was Not Bootstrapped With A Fleet Server Panics
.  {Fleet-Server-issue}https://github.com/elastic/elastic-agent/issues/2170[#https://github.com/elastic/elastic-agent/issues/2170]
[%collapsible]
====

====
osquery::

[discrete]
[[known-issue-issue-https://github.com/elastic/elastic-agent/issues/2066]]
.Osquery Live Query Results Can Take Up To 5 Mins To Show Up In Kibana
. {osquery-pull}https://github.com/elastic/elastic-agent/pull/2076[#https://github.com/elastic/elastic-agent/pull/2076] {osquery-issue}https://github.com/elastic/elastic-agent/issues/2066[#https://github.com/elastic/elastic-agent/issues/2066]
[%collapsible]
====

====








[discrete]
[[bug-fixes-8.6.1]]
=== Bug fixes


elastic-agent::

* Fix Issue Where Beats Started By Agent May Fail With An &#39;Output Unit Has No Config&#39; Error. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2138[#https://github.com/elastic/elastic-agent/pull/2138] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/2086[#https://github.com/elastic/elastic-agent/issues/2086]
* Fix Broken Deployment Using Traffic Filter By Injecting Custom Headers. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2158[#https://github.com/elastic/elastic-agent/pull/2158] {elastic-agent-issue}https://github.com/elastic/beats/issues/32993[#https://github.com/elastic/beats/issues/32993]
* Allow Easily Filtering Only The Agent Logs From The Combined Agent Log Gile. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2044[#https://github.com/elastic/elastic-agent/pull/2044] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/1810[#https://github.com/elastic/elastic-agent/issues/1810]

// end 8.6.1 relnotes

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 3]
```diff
@@ -14,13 +14,86 @@
...
+* Fix package installation APIs to install packages without a version {kibana-pull}149193[#149193]
+* Fix issue where the latest GA version could not be installed if there was a newer prerelease version in the registry
+{kibana-pull}149133[#149133] {kibana-pull}149104[#149104]
+
+{fleet-server}::
+* No bug fixes in this release
+
+{agent}::
+* Fix issue where {beats} started by {agent} may fail with an `output unit has no config` error {agent-pull}2138[#2138] {agent-issue}2086[#2086]
+* Fix broken deployments that use traffic filter by injecting custom headers {agent-pull}2158[#2158] {beats-issue}32993[#32993]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:92]: I still don't completely like the structure of this description (it's grammatically vague), but I think it's OK?

Comment (cmacknz) [reply docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:92]: We can reword this to something like "Restore the ability to set custom HTTP headers at enrolment time. Fixes traffic filters in Integrations Server cloud deployments."

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 4]
```diff
@@ -14,13 +14,86 @@
...
+=== Bug fixes
+
+{fleet}::
+* Fix missing policy Id in installation URL for cloud integrations {kibana-pull}149243[#149243]
+* Fix package installation APIs to install packages without a version {kibana-pull}149193[#149193]
+* Fix issue where the latest GA version could not be installed if there was a newer prerelease version in the registry
+{kibana-pull}149133[#149133] {kibana-pull}149104[#149104]
+
+{fleet-server}::
+* No bug fixes in this release
```
Comment (cmacknz) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:88]: There was a missing backport we added to 8.6.1 that fixes a bug with the Logstash output: https://github.com/elastic/fleet-server/commit/759ba44fd471fbb084b3d4e1b6d77e4823572ddc

Looks like we missed adding a changelog entry for the original fix, but we can add one here. Something like:

"Update the .fleet-agent index when acknowledging policy changes when Logstash is the configured output. Fixes agents always showing as Updating when using the Logstash output."

**Review by cmacknz — APPROVED** [review]
